app/src/main/java/com/keylesspalace/tusky/components/conversation/ConversationsFragment.kt

```diff
@@ -87,10 +85,10 @@ class ConversationsFragment : SFragment(), StatusActionListener, Injectable, Res
 
         initSwipeToRefresh()
 
-        viewModel.conversations.observe(this, Observer<PagedList<ConversationEntity>> {
+        viewModel.conversations.observe(viewLifecycleOwner, Observer<PagedList<ConversationEntity>> {
```

Comment:
Why did this change?

Reply:
its a new inspection that suggest to do so